openedx/core/djangoapps/ccxcon/tests/test_tasks.py

```diff
@@ -0,0 +1,47 @@
+"""
+Tests for the CCXCon celery tasks
+"""
+
+import mock
+
+from django.test import TestCase
+
+from opaque_keys.edx.keys import CourseKey
+from openedx.core.djangoapps.ccxcon import api, tasks
+
+
+class CCXConTaskTestCase(TestCase):
+    """
+    Tests for CCXCon tasks.
+    """
+
+    @mock.patch('openedx.core.djangoapps.ccxcon.api.course_info_to_ccxcon')
+    def test_update_ccxcon_task_ok(self, mock_citc):
+        """
+        Test task with no problems
+        """
+        mock_response = mock.Mock()
+        mock_citc.return_value = mock_response
+
+        course_id = u'course-v1:OrgFoo+CN199+CR-FALL01'
+        result = tasks.update_ccxcon.delay(course_id)
+
+        mock_citc.assert_called_once_with(CourseKey.from_string(course_id))
+
+
+    @mock.patch('openedx.core.djangoapps.ccxcon.api.course_info_to_ccxcon')
+    def test_update_ccxcon_task_retry(self, mock_citc):
+        """
+        Test task with exception that triggers a retry
+        """
+        mock_citc.side_effect = api.CCXConnServerError()
+        course_id = u'course-v1:OrgFoo+CN199+CR-FALL01'
+        result = tasks.update_ccxcon.delay(course_id)
+
+        self.assertEqual(mock_citc.call_count, 6)
+        course_key = CourseKey.from_string(course_id)
+        for call in mock_citc.call_args_list:
+            c_args, c_kwars = call
```

Comment:
Can you rename to `c_kwargs` if that's what it's referring to? I'm also unsure about the edX rules on variable names but it seems like just `args` and `kwargs` would be cleaner.